gateway/1.4/modules/ROOT/pages/_partials/task-tls-config.adoc

```diff
@@ -249,8 +250,100 @@ spec:
 ----
 //end::sample-config-mtls[]
 
+//tag::sample-config-all-outbound-intro[]
+*Sample configuration for adding outbound TLS context for all API Instances running on this Flex Gateway:*
+//end::sample-config-outbound-all-intro[]
+//tag::sample-config-outbound-all[]
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com/v1alpha1
+kind: PolicyBinding
+metadata:
+  name: ingress-https-tls
+spec:
+  targetRef:
+    kind: Selector 
+    selector:
+      kind: ApiInstance
+  policyRef:
+    name: tls
+  config:
+    certificate:
+      key: |
+        # -----BEGIN PRIVATE KEY-----
+        # insert certificate key
+        # -----END PRIVATE KEY-----
+      crt: |
+        # -----BEGIN CERTIFICATE-----
+        # insert certificate
+        # -----END CERTIFICATE-----
+    alpn:
+      - http/1.1
+      - h2
+    minversion: "1.1"
+    maxversion: "1.3"
+    ciphers:
+      - TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+      - TLS_CHACHA20_POLY1305_SHA256
```

Comment:
double check this list. At least TLS_CHACHA20_POLY1305_SHA256 should not be here (it's a 1.3 cipher)